eventhubs/spring-cloud-azure-starter/spring-cloud-azure-sample-eventhubs-kafka/README.md

```diff
@@ -1,121 +1,161 @@
-# Spring Cloud Azure Sample Stream Event Hubs Kafka
+# Spring Cloud Azure Sample Stream Event Hubs Kafka 
 
-## Key concepts
-
-This code sample demonstrates how to use the Spring Cloud Azure Starter and Spring Cloud Starter Stream Kafka for Azure Event Hubs. The sample app exposes a RESTful API to receive
+This sample demonstrates how to use the Spring Cloud Azure Starter and Spring Cloud Starter Stream Kafka for Azure Event Hubs. The sample app exposes a RESTful API to receive
 string message. Then message is sent through Azure Event Hubs to a bean `consumer`
 which simply logs the message.
 
-## Getting started
-
-
-Running this sample will be charged by Azure. You can check the usage and bill at
-[this link][azure-account].
-
-
-### Create Azure resources
-
-1. Create a service principal for use in by your app. Please follow 
-   [create service principal from Azure CLI][create-sp-using-azure-cli]. 
-   The credential is not required since Spring Cloud Azure support https://docs.microsoft.com/en-us/azure/developer/java/sdk/identity,
-   you only need to log in with az cli / vs code or Intellij Azure Toolkit, then credential information will be left out of properties
-
-3. Create [Azure Event Hubs][create-event-hubs]. Please notice that Event Hubs for Kafka is only supported for [Standard and Dedicated tier namespaces](https://azure.microsoft.com/pricing/details/event-hubs/).
-
-## Examples
-
-### Use Event Hubs connection string
-
-The simplest way to connect to Event Hubs for Kafka is with the connection string. 
-
-1. Update
-    [application.yaml][application.yaml]
-    file
-    
-    ```yaml
-    spring:
-      cloud:
-        azure:
-          eventhubs:
-            connection-string: ${AZURE_EVENTHUBS_CONNECTION_STRING}
-        stream:
-          function:
-            definition: consume;supply
-          bindings:
-            consume-in-0:
-              destination: sample-eventhubs-kafka
-              group: $Default
-            supply-out-0:
-              destination: sample-eventhubs-kafka
-    ```
-
-2. Run the `mvn spring-boot:run` in the root of the code sample to get the app running.
-
-3. Send a POST request
-
-        $ curl -X POST http://localhost:8080/messages?message=hello
-
-4. Verify in your app’s logs that a similar message was posted:
-
-    `New message received: hello`
-
-5. Delete the resources on [Azure Portal][azure-portal] to avoid unexpected charges.
-
-### Use Azure Resource Manager to retrieve connection string
```

Comment:
Should we have another configuration file to demonstrate this case?

Reply:
Tracked in this issue: https://github.com/Azure/azure-sdk-for-java/issues/26407